### PR TITLE
ua-i update /server

### LINKED ISSUE #5019: [UA-I] Update /server page
[copy doc](https://docs.google.com/document/d/1vxLhtbjgPs9PCNEhr7fbhEoErCPMXYFfU4UTKH7Kbq4/edit)

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Update /server for UA-I

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/server](http://0.0.0.0:8001/server)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1vxLhtbjgPs9PCNEhr7fbhEoErCPMXYFfU4UTKH7Kbq4/edit#heading=h.6alb6mc7cvu9)

## Issue / Card

Fixes #5019

### CODE REVIEW

File: templates/server/index.html [hunk 1]
```diff
@@ -169,59 +159,114 @@ <h3 class="p-muted-heading u-align--center">Works with all your hardware and sof
...
   <div class="row">
     <div class="col-6">
-      <p>The Ubuntu Advantage service programme provides fast problem resolution, direct access to Ubuntu experts and efficient administration with the Ubuntu systems management package, Landscape. Available in Standard (business hours) and Advanced (24x7) support.</p>
-      <p>UA for Infrastructure, Advanced includes Server, OpenStack and Kubernetes support.</p>
-      <p><a href="/support">Learn more about Ubuntu Advantage for Infrastructure&nbsp;&rsaquo;</a></p>
+      <p>
+        Ubuntu Advantage for infrastructure offers a single, per-node packaging of the most comprehensive software, security and IaaS support in the industry. With OpenStack and Kubernetes support included, Ubuntu Advantage for Infrastructure provides everything you need to future-proof your data centre.
+      </p>
+      <p>
+        UA for Infrastructure, Advanced includes Server, OpenStack and Kubernetes support. and is available at Standard (24x5) and Advanced (24x7) response times.
```
Comment (anthonydillon) [inline templates/server/index.html]: Copy doc has "UA for Infrastructure, Advanced includes Server, OpenStack and Kubernetes support and is available at Standard (24x5) and Advanced (24x7) response times."

File: templates/server/index.html [hunk 2]
```diff
@@ -42,85 +52,63 @@ <h2>What&rsquo;s new in {{lts_release_full}}</h2>
         <li class="p-list__item is-ticked">Certification as a guest on AWS, Microsoft Azure, Joyent, IBM, Google Cloud Platform and Rackspace</li>
         <li class="p-list__item is-ticked">Easily configure networking through netplan.io</li>
       </ul>
-      <p><a class="p-link--external"
-        href="https://wiki.ubuntu.com/{{lts_release_name}}/ReleaseNotes">Read more in the release notes
-      </a></p>
-    </div>
+      <p>
```
[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/server/index.html]: that's there, the order is just different, I made the order the same now

[the remark this answers is not in the document]
Comment (anthonydillon) [reply templates/server/index.html]: NGINX is (v1.14)

**Comment by anthonydillon** [comment]
Just a comment on the version of NGINX

File: templates/server/index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/server/index.html]: updated

**Comment by anthonydillon** [comment]
LGTM :+1:

**Pull request merged**